java/runtime/src/main/java/io/ray/runtime/functionmanager/FunctionManager.java

```diff
@@ -192,9 +193,11 @@ RayFunction getFunction(JavaFunctionDescriptor descriptor) {
     }
 
     /** Load all functions from a class. */
-    Map<Pair<String, String>, RayFunction> loadFunctionsForClass(String className) {
+    Map<Pair<String, String>, Pair<RayFunction, Boolean>> loadFunctionsForClass(String className) {
       // If RayFunction is null, the function is overloaded.
-      Map<Pair<String, String>, RayFunction> map = new HashMap<>();
+      // The value of this map is a pair of <rayFunction, isDefault>.
+      // The `isDefault` is used to mark if the method is a marked as default keyword.
+      Map<Pair<String, String>, Pair<RayFunction, Boolean>> map = new HashMap<>();
```

Comment:
Not necessary to add a default field using `Pair`, just check whether the method is default when needed?

Reply:
`isDefault` is not used for the overrided method in child class in Java. It's not able to check it when we need it.